UnityProject/Assets/Scripts/InGameEvents/InGameEventScripts/EventGiveGuns.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace InGameEvents
+{
+	public class EventGiveGuns : EventScriptBase
+	{
+
+		public List<GameObject> GunList = new List<GameObject>();
```

Comment:
I don't think public is necessary here. Use private with [SerializeField] attribute instead.

Reply:
Done